torch/_inductor/codegen/triton.py

```diff
@@ -2850,6 +2850,9 @@ def select_tiling(cls, node_schedule, numel, reduction_numel=sympy.Integer(1)):
     def flush(self):
         pass
 
+    def ready_to_flush(self) -> bool:
+        return False
```

Comment:
It is same as `BaseScheduling`, no need to define it again here?

Reply:
> It is same as `BaseScheduling`, no need to define it again here?

I think leave a API to triton, help further developer understand code.